scripts/run-registry.ts

```diff
@@ -146,25 +136,31 @@ const run = async () => {
 
   logger.log(`🌿 verdaccio running on ${verdaccioUrl}`);
 
-  // in some environments you need to add a dummy user. always try to add & catch on failure
-  try {
-    await addUser(verdaccioUrl);
-  } catch (e) {
-    //
-  }
+  logger.log(`👤 add temp user to verdaccio`);
+  await execa(
+    'npx',
+    // creates a .npmrc file in the root directory of the project
+    ['npm-auth-to-token', '-u', 'foo', '-p', 's3cret', '-e', 'test@test.com', '-r', verdaccioUrl],
+    {
+      cwd: root,
+    }
+  );
 
   logger.log(`📦 found ${packages.length} storybook packages at version ${chalk.blue(version)}`);
 
   if (program.publish) {
     await publish(packages, verdaccioUrl);
   }
 
+  await execa('npx', ['rimraf', '.npmrc'], { cwd: root });
```

Comment:
We should just use a `fs` node API here, rather than go through `exaca>child_process>npx>rimraf`.
That seems a lot just to delete a single file.

Reply:
Sounds reasonable! I will do the adjustment tomorrow.